stylesheets/winter-sports.oss

```diff
@@ -499,7 +499,7 @@ STYLE
     ICON { name: target;} */
 
   [MAG world-] {
-    [TYPE _route] WAY {color: @routeColor; displayWidth: 1.5mm; priority: 100; }
+    [TYPE _route] WAY {color: @routeColor; displayWidth: 1.5mm; width: 6m; priority: 100;}
```

Comment:
Would not be better to use `MODULE "route"` ? I know that it is one-liner, but still, we may de-duplicate stylesheets a bit.